DigitalOffice/Services/TimeManagementService/Validators/CreateWorkTimeRequestValidator.cs

```diff
@@ -0,0 +1,87 @@
+﻿using FluentValidation;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using LT.DigitalOffice.TimeManagementService.Models;
+using LT.DigitalOffice.TimeManagementService.Repositories.Filters;
+using LT.DigitalOffice.TimeManagementService.Repositories.Interfaces;
+using System.Globalization;
+
+namespace LT.DigitalOffice.TimeManagementService.Validators
+{
+    public class CreateWorkTimeRequestValidator : AbstractValidator<CreateWorkTimeRequest>
+    {
+        private IWorkTimeRepository repository;
+
+        /// <summary>
+        /// How many days ago can WorkTime be added.
+        /// </summary>
+        public const int fromDay = 3;
+        /// <summary>
+        /// How many days ahead can WorkTime be added.
+        /// </summary>
+        public const int toDay = 2;
+        /// <summary>
+        /// Limit on working hours in a row.
+        /// </summary>
+        public readonly static TimeSpan workingLimit = new TimeSpan(24, 0, 0); // 24 hours.
+
+        private readonly DateTime fromDateTime = DateTime.Now.AddDays(-fromDay);
+        private readonly DateTime toDateTime = DateTime.Now.AddDays(toDay);
+        private readonly CultureInfo culture = CultureInfo.CreateSpecificCulture("en-GB");
+
+        public CreateWorkTimeRequestValidator([FromServices] IWorkTimeRepository repository)
+        {
+            this.repository = repository;
+
+            RuleFor(wt => wt.WorkerUserId)
+                    .NotEmpty();
+
+            RuleFor(wt => wt.StartTime)
+                .NotEqual(new DateTime())
+                .Must(st => st > fromDateTime)
+                .WithMessage(date => $"WorkTime had to be filled no later than {fromDateTime.ToString(culture)}.")
+                .Must(st => st < toDateTime)
+                .WithMessage(date => $"WorkTime cannot be filled until {toDateTime.ToString(culture)}.");
```

Comment:
Also I'm confused about these messages.
I think you mixed up the first and the second messages.
Like, you can't fill later than ToDateTime (the latest already)
and you can't fill earlier that FromDateTime.

In current state, messages show:
You have to fill time before FromDateTime
You Have to fill time after ToDateTime.

Please, check this. I might be mistaken.

Reply:
No. 
no later than => more than start
until => less then end